dongle_app/js/app.js

```diff
@@ -151,4 +151,23 @@ window.onload = function() {
       showQR(dongle);
     }
   );
+
+  function updateOnlineStatus(event) {
```

Comment:
If you want to add even more options to the user, check [1]. With this you can forward the user to the Settings App in order to enable data connection! This is a nice to have, not a blocker.

[1] https://github.com/mozilla-b2g/firefoxos-loop-client/blob/master/app/js/screens/error_screen.js#L36,